src/components/admin/navigation/AdminLayout.tsx

```diff
@@ -0,0 +1,225 @@
+
+import PanelFooter from './PanelFooter'
+import { menuItems } from './adminMenu'
+import CustomListItem from './CustomListItem'
+
+const drawerWidth = 200
+const useStyles = makeStyles({
+  drawerHeader: {
+    width: drawerWidth,
+    height: 64,
+    position: 'absolute',
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'space-between',
+    padding: '0 19px 0 24px',
+  },
+  wrapper: {
+    display: 'flex',
+    position: 'relative',
+    minHeight: '100vh',
+    paddingRight: '24px',
+  },
+  appbarHeader: {
+    width: `calc(100% - ${drawerWidth}px)`,
+    height: 64,
+    marginLeft: '6rem',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+  appbarWrapper: {
+    display: 'flex',
+    width: 'calc(100% - 24px)',
+    position: 'relative',
+    paddingRight: '16px',
+  },
+  logoWrapper: {
+    width: 150,
+    display: 'flex',
+  },
+  logo: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+})
+
+const openedMixin = (theme: Theme): CSSObject => ({
+  width: drawerWidth,
+  transition: theme.transitions.create('width', {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.enteringScreen,
+  }),
+  border: 'none',
+  overflowX: 'hidden',
+})
+
+const closedMixin = (theme: Theme): CSSObject => ({
+  transition: theme.transitions.create('width', {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.leavingScreen,
+  }),
+  overflowX: 'hidden',
+  border: 'none',
+  width: `calc(${theme.spacing(7)} + 1px)`,
+  [theme.breakpoints.up('sm')]: {
+    width: `calc(${theme.spacing(9)} + 1px)`,
+  },
+})
+
+// @ts-expect-error bad MUI types for styled.div
+const DrawerHeader = styled('div')(({ theme }: { theme: Theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'flex-end',
+  padding: theme.spacing(0, 1),
+  // necessary for content to be below app bar
+  ...theme.mixins.toolbar,
+}))
+
+interface AppBarProps extends MuiAppBarProps {
+  open?: boolean
+}
+
+const AppBar = styled(MuiAppBar, {
+  shouldForwardProp: (prop: string) => prop !== 'open',
+})<AppBarProps>(({ theme }: { theme: Theme }) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  background: 'none',
+  boxShadow: 'none',
+  position: 'fixed',
+  zIndex: theme.zIndex.drawer + 1,
+  height: '64px',
+  width: `100%`,
+}))
+
+const Drawer = styled(MuiDrawer, { shouldForwardProp: (prop: string) => prop !== 'open' })(
+  ({ theme, open }: { theme: Theme; open: boolean }) => ({
+    width: drawerWidth,
+    flexShrink: 0,
+    whiteSpace: 'nowrap',
+    boxSizing: 'border-box',
+    ...(open && {
+      ...openedMixin(theme),
+      '& .MuiDrawer-paper': openedMixin(theme),
+    }),
+    ...(!open && {
+      ...closedMixin(theme),
+      '& .MuiDrawer-paper': closedMixin(theme),
+    }),
+  }),
+)
+
+type Props = {
+  children: React.ReactNode
+}
+
+export default function AdminLayout({ children }: Props) {
```

Comment:
@Marina-yoya  Please reuse the already existing `AdminLayout` from the `master` branch

Maybe you'll need to merge master to your branch in order to get it.

Ping me if you need any assistance along the way